### PR TITLE
UX: Give fa-hearts more love.

### PR DESCRIPTION
**Pull request by tgxworld**
Follow up to https://github.com/discourse/discourse/pull/3548.

### CODE REVIEW

**Comment by tgxworld** [comment]
@coding-horror I changed the color of the heart icon in notifications as well. Please let me know if that is ok :)

File: app/assets/stylesheets/common/admin/admin_base.scss [hunk 1]
```diff
@@ -773,6 +773,9 @@ table.api-keys {
       i.fa {
         color: $primary;
       }
+      i.fa-heart {
+        color: $love;
+      }
```
Comment (tgxworld) [inline app/assets/stylesheets/common/admin/admin_base.scss]: screenshot from 2015-07-22 19 05 44

File: app/assets/stylesheets/common/base/header.scss [hunk 2]
```diff
@@ -193,6 +193,7 @@
   // Notifications
   &#notifications-dropdown {
     .fa { color: scale-color($primary, $lightness: 50%); }
+    .fa-heart { color: $love; }
```
Comment (tgxworld) [inline app/assets/stylesheets/common/base/header.scss]: screenshot from 2015-07-22 19 10 38
screenshot from 2015-07-22 19 10 16

File: app/assets/stylesheets/common/base/user.scss [hunk 3]
```diff
@@ -15,6 +15,12 @@
   }
 }
 
+.user-main {
+  i.fa-heart {
+    color: $love !important;
+  }
+}
```
Comment (tgxworld) [inline app/assets/stylesheets/common/base/user.scss]: screenshot from 2015-07-22 23 48 17
screenshot from 2015-07-22 23 39 32
screenshot from 2015-07-22 19 18 34
screenshot from 2015-07-22 19 18 16
screenshot from 2015-07-22 19 17 34
screenshot from 2015-07-22 19 17 27

**Comment by coding-horror** [comment]
looks good let's try it again

**Pull request merged**